tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md

```diff
@@ -1,36 +1,43 @@
-Each project starts with the default pipeline specified in
+Each Semaphore 2.0 project starts with the default pipeline specified in
 `.semaphore/semaphore.yml`. Real world pipelines tend to branch out
-when certain conditions are met. Examples may be deploying to
-production on master builds or deploying to a pre-production environment
-on topic branches.
+when certain conditions are met. Examples may be deploying to production on
+master builds or deploying to a pre-production environment on topic branches.
 
-On Semaphore deployment and delivery is managed with _promotions_, which may
-be automatic or manual and optionally depend on conditions.
+On Semaphore 2.0, deployment and delivery is managed with _promotions_, which
+may be performed automatically or manually and may optionally depend on
+user-defined conditions.
 
 ## Manual deployment
 
 Let's start by adding a manual confirmation to promote to production.
 
-<pre><code class="language-yaml"># .semaphore/semaphore.yml
```

Comment:
@mactsouk the reason why this was in one line is that if the first line of code snippet starts from line below, it creates too much white space on the live web page.

Reply:
Yes, but this way leaves a space character in front of the first line.